NYPLAudiobookToolkit/UI/AudiobookPlayerViewController.swift

```diff
@@ -406,6 +530,7 @@ extension AudiobookPlayerViewController: AudiobookNetworkServiceDelegate {
     public func audiobookNetworkService(_ audiobookNetworkService: AudiobookNetworkService, didUpdateDownloadPercentageFor spineElement: SpineElement) {}
     public func audiobookNetworkService(_ audiobookNetworkService: AudiobookNetworkService, didDeleteFileFor spineElement: SpineElement) {}
     public func audiobookNetworkService(_ audiobookNetworkService: AudiobookNetworkService, didReceive error: NSError, for spineElement: SpineElement) {
+        //TODO
```

Comment:
what is `//TODO`?

Reply:
My mistake! Eventually I'm going to let a host provide a closure that AudiobookToolkit can use to forward logs that the host would be interested in collecting.. for analytics in our case, but whatever reason.